service/app/commands/message_buffer.go

```diff
@@ -128,6 +128,15 @@ func (m *MessageBuffer) persistTransaction(adapters Adapters, feedsToPersist fee
 		authorRef := firstMessage.Author()
 		feedRef := firstMessage.Feed()
 
+		feedIsBanned, err := adapters.BanList.ContainsFeed(feedRef)
+		if err != nil {
+			return errors.Wrap(err, "error checking if the feed is banned")
+		}
+
+		if feedIsBanned {
+			return errors.New("feed is banned")
+		}
```

Comment:
Is it worth failing the whole command if just one feed is banned? It would be lame to make persisting of all the messages fail because of one feed being banned 🤔 

If it's just one of these errors just in case of code logical errors and this situation is expected never to happen it's OK to leave it like this I guess.

Reply:
It is totally lame and I think some fixes for this will come in here #44 or later. I just decided not to deal with this now.